src/pages/Kimhojoon/Login/LoginKhj.js

```diff
@@ -1,6 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './LoginKhj.scss';
+import { useNavigate } from 'react-router-dom';
+import { isDisabled } from '@testing-library/user-event/dist/utils';
 
 export default function LoginKhj() {
-  return <div>Login_khj</div>;
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/MainKhj');
+  };
+
+  const [idValue, setIdValue] = useState('');
+  const saveUserId = e => {
+    setIdValue(e.target.value);
+  };
+
+  const [pwValue, setPwValue] = useState('');
+  const saveUserPw = e => {
+    setPwValue(e.target.value);
+  };
+
+  // const enter = e => {
+  //   if (e.code == 'Enter') {
+  //     goToMain();
+  //   }
+  // };
+
+  return (
+    <div className="container">
+      <div className="westagram">Westagram</div>
+      <div className="userinfo">
+        <div className="id2">
+          <input
+            className="inputid"
+            type="text"
+            placeholder="전화번호, 사용자 이름 또는 이메일"
+            onChange={saveUserId}
+          />
+        </div>
+        <div className="password">
+          <input
+            className="inputpassword"
+            type="password"
+            placeholder="비밀번호"
+            onChange={saveUserPw}
+            // onKeyUp={enter} 활성화가 안되어잇어도 엔터누르면 넘어감
+          />
+        </div>
+      </div>
+
+      <button
+        className="loginbutton"
+        onClick={goToMain}
+        type="button"
+        disabled={idValue.includes('@') && pwValue.length >= 5 ? false : true}
```

Comment:
- 유효성 검사는 disabled의 값과 opacity의 값을 결정하는 곳, 두곳에서 필요하니 로직을 따로빼서 관리해주세요!
- opacity값 변경 로직도 추가해주세요

Reply:
const isInputValid = idValue.includes('@') && pwValue.length >= 5;

```suggestion
        disabled={!isInputValid}
```